Patches/Core/ThingDefs_Races/Races_Mechanoid.xml

```diff
@@ -7,7 +7,7 @@
 		<xpath>Defs/ThingDef[@Name="BaseMechanoid"]/statBases</xpath>
 		<value>
 			<SmokeSensitivity>0</SmokeSensitivity>
-			<NightVisionEfficiency>1</NightVisionEfficiency>
+			<NightVisionEfficiency>0.80</NightVisionEfficiency>
```

Comment:
Maybe 0.7? Since power armor has 0.7. Or we can make assign a different one for each mech type based on their fire rate ranging from 0.5 to 0.8.

Reply:
In the lore, mechs are advanced spacer or glitterworld tech, so I figured it made sense to give them somewhat better than average NV.